AddressApp.Tests.Unit/DeleteMe.cs

```diff
@@ -8,8 +8,8 @@ namespace AddressApp.Tests.Unit
 {
     public class DeleteMe
     {
-        [Fact]
+        [Fact]  
```

Comment:
Please run code cleanup to remove unwanted spaces.

Reply:
@ShriHumrudha  I have run the code cleanup. Please check and let me know your feedback.